plugins/reach/lib/model/EntryVendorTask.php

```diff
@@ -453,49 +453,68 @@ public function postSave(PropelPDO $con = null)
 
 	public function addSchedulingData()
 	{
+		$language = '';
```

Comment:
dont really see a reason to define it here, you already allocate it in both places you use it